README.md

```diff
@@ -13,41 +13,73 @@ GLUON_SITE_FEEDS='community'
 
 PACKAGES_COMMUNITY_REPO=https://github.com/freifunk-gluon/community-packages.git
 PACKAGES_COMMUNITY_COMMIT=<COMMIT-HASH>
+PACKAGES_COMMUNITY_BRANCH=master
 ```
 
-## Commit guidelines
+Your commit has to be on the branch specified in `PACKAGES_COMMUNITY_BRANCH`.
+If you leave out the `PACKAGES_COMMUNITY_BRANCH`, the repository default branch (master) is used.
```

Comment:
That's not correct anymore. The default branch selected on GitHub is being used, not master.

Reply:
sure, whatever the owner of the repository sets as repository default branch - but this is the master branch (currently).
If we change this, we need to adjust the docs too, anyway - as this is written before too?

Reply:
I meant do be more clarifying. I can remove the brackets too

Reply:
I think removing the brackets makes most sense.

Reply:
Thanks!